tensorflow/compiler/xla/service/cpu/cpu_runtime_avx.h

```diff
@@ -46,6 +47,7 @@ xla::cpu::runtime::V8F32AVX __xla_cpu_runtime_ExpV8F32AVX(
 
 xla::cpu::runtime::V8F32AVX __xla_cpu_runtime_LogV8F32AVX(
     xla::cpu::runtime::V8F32AVX x) TF_ATTRIBUTE_WEAK;
```

Comment:
With this change we should no longer need the `TF_ATTRIBUTE_WEAK` right?

Reply:
Yes. I will remove TF_ATTRIBUTE_WEAK in my next update.